searches/linear_search.py

```diff
@@ -38,6 +38,41 @@ def linear_search(sequence, target):
     return None
```

Comment:
Line 14 promises that we are going to return an int but line 38 returns None.  We should not surprise the caller.